src/services/pay-services.ts

```diff
@@ -18,4 +18,21 @@ export default class PayServices {
       .then(response => response?.data)
       .catch(() => {}) // ignore errors
   }
+
+  /**
+   * Fetches the CFS account ID from the pay-api.
+   * @param payApiUrl the URL of the pay-api
+   * @param orgId the organization ID for which to fetch the CFS account ID
+   * @returns the CFS account ID
+   */
+  static async fetchCfsAccountId (payApiUrl: string, orgId: number): Promise<string> {
+    const url = `${payApiUrl}accounts/${orgId}`
+    try {
+      const response = await axios.get(url)
+      return response?.data.cfsAccount.cfsAccountNumber
```

Comment:
How does this handle if you manually override the payment method to non CFS?

`response?.data?.cfsAccount?.cfsAccountNumber` <- I'd recommend using incase someone is forcefully switched (unlikely)